### PR TITLE
remove btc pub key; remove keeper NFT in DeCusSystem

### PR DESCRIPTION
**Pull request by zimengpan**

### CODE REVIEW

File: contracts/keeper/GroupLib.sol [hunk 1]
```diff
@@ -36,6 +36,21 @@ library GroupLib {
         return _map.groups[_map.id2index[_id] - 1].currSatoshi == 0;
     }
 
+    function isGroupKeeper(
+        GroupMap storage _map,
+        uint256 _id,
+        uint256 _keeperID
+    ) internal view returns (bool) {
+        uint256[] memory keepers = _map.groups[_map.id2index[_id] - 1].keepers;
```
Comment (fudorec) [inline contracts/keeper/GroupLib.sol:44]: Just curious, memory or storage, which one cost less gas?

File: contracts/core/DeCusSystem.sol [hunk 2]
```diff
@@ -83,9 +79,9 @@ contract DeCusSystem is AccessControl, Pausable {
         _mintToUser(_groupId);
     }
 
-    function revokeMintRequest(uint256 _groupId, uint256 _tokenId) public {
+    function revokeMintRequest(uint256 _groupId, uint256 _keeperID) public {
```
Comment (fudorec) [inline contracts/core/DeCusSystem.sol:82]: I like the removal of keeperNFT here

Comment (fudorec) [reply contracts/core/DeCusSystem.sol:82]: Later we might consider replace groupId with receipt Id

**Review by fudorec — APPROVED** [review]

File: contracts/keeper/GroupLib.sol (hunk 1 above)
Comment (zimengpan) [reply contracts/keeper/GroupLib.sol:44]: memory in general

**Pull request merged**